packages/terra-form-checkbox/src/CheckboxField.module.scss

```diff
@@ -84,4 +84,16 @@
     margin-top: var(--terra-form-checkbox-field-error-text-margin-top, 0.357em);
     text-align: left;
   }
+
+  .visually-hidden {
```

Comment:
curious, where is this style being applied?

Reply:
Refactored some code and forgot to remove this. Removed in this commit: https://github.com/cerner/terra-core/pull/2293/commits/51e5ef65c709a651e4616af1b6ec6f6c984562ee